sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -33,9 +33,9 @@ public class ServiceBusProcessor : IAsyncDisposable
 
         private Func<ProcessErrorEventArgs, Task> _processErrorAsync;
 
-        private Func<ProcessSessionEventArgs, Task> _sessionInitializingAsync;
+        internal Func<ProcessSessionEventArgs, Task> _sessionInitializingAsync;
```

Comment:
If these are intended to be raised via the `OnXXX` methods, should we consider keeping them `private`?

Reply:
I made this internal due to some refactoring where I only pass the processor to the receiverManager.